targets/particle/source/main.cpp

```diff
@@ -70,6 +70,7 @@ js_delay (const jerry_value_t func_value, /**< function object */
 static void
 init_jerry ()
 {
+  srand ((unsigned) (jerry_port_get_current_time ()));
```

Comment:
I cannot find which port implementation the particle target is using. Does it have a sensible `jerry_port_get_current_time` implementation? (There is a chance that the target went bitrotten. If that's the case, we should not try touch it up now with `srand` but let a target maintainer bring it up to speed first -- or add `srand` at the same time.)